src/Avalonia.Controls/TreeViewItemExpansionChangedEventArgs.cs

```diff
@@ -0,0 +1,24 @@
+using Avalonia.Interactivity;
+
+namespace Avalonia.Controls;
+
+/// <summary>
+/// 
+/// </summary>
+public class TreeViewItemExpansionChangedEventArgs: RoutedEventArgs
+{
+    public TreeViewItem Source { get; }
```

Comment:
This property can be removed for consistency and not conflict with `object RoutedEventArgs.Source`

Reply:
Actually, with events separation, this custom event args are less useful now.